docs/01-declarations.md

```diff
@@ -94,3 +94,13 @@ declare module "path/to/M" {
   ...
 }
 {% endhighlight %}
+
+## Modules that export single function
```

Comment:
So this feature isn't necessarily specific to functions -- it works with any value (classes, objects, numbers, etc). It's also specific to CommonJS modules.

Maybe you could clarify this to `CommonJS modules with a single export`?